src/com.company/heap/BClosestPointToOrigin.java

```diff
@@ -0,0 +1,105 @@
+package heap;
+
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.PriorityQueue;
+
+public class BClosestPointToOrigin {
+
+    private int[][] findBClosestPointToOrigin(int[][] points, int b) {
+        PriorityQueue<int[]> priorityQueue = new PriorityQueue<>(new Comparator<int[]>() {
+            @Override
+            public int compare(int[] o1, int[] o2) {
+                double distance1 = Math.pow(o1[0] - 0, 2) + Math.pow(o1[1] - 0, 2);
+                double distance2 = Math.pow(o2[0] - 0, 2) + Math.pow(o2[1] - 0, 2);
+                return Double.compare(distance1, distance2);
+            }
+        });
+
+        for (int i = 0; i < points.length; i++) {
+            priorityQueue.add(points[i]);
+        }
+        int[][] result = new int[1][2];
+        result[0] = priorityQueue.poll();
+        return result;
+    }
+
+    public static void main(String[] args) {
+        int[][] points = new int[][]{{1, 3}, {-2, 2}};
+        int b = 1;
```

Comment:
Why do you hardcode the value of b to 1? Make the input variables parameterized by writing tests, so that you can test across multiple test cases.